backend/server/src/admin.rs

```diff
@@ -23,3 +24,14 @@ pub async fn get(user: User, db: Database) -> Json<AdminInfoResponse> {
             .await,
     })
 }
+
+#[post("/make_superuser", data="<email>")]
+pub async fn make_superuser(_user: SuperUser, db: Database, email: Json<String>) -> Result<(), JsonErr<()>> {
+    db.run(move |conn| {
+        User::get_from_email(conn, &email.into_inner())
+            .map(|user| user.make_superuser(conn).ok())
+            .flatten()
+    })
+    .await
+    .ok_or(JsonErr((), Status::BadRequest))
+}
```

Comment:
does this just let anyone make superusers

Reply:
Nope, need to already be a superuser. The _user: SuperUser parameter does that check for you epicly

Reply:
ahh i see